docs/src/main/sphinx/connector/hive.rst

```diff
@@ -834,6 +834,17 @@ information.
 
 Refer to :doc:`the migration guide </appendix/from-hive>` for practical advice on migrating from Hive to Trino.
 
+ALTER TABLE
+^^^^^^^^^^^
+
+.. _hive-set-column-type:
+
+The connector supports changing column types with
+``ALTER TABLE [ IF EXISTS ] name ALTER COLUMN column_name SET DATA TYPE new_type``.
+
+The column change command will only modify Hive's metadata, and will not modify data.
+Users should make sure the actual data layout of the table/partition conforms with the metadata definition.
```

Comment:
I don't understand the last sentence.

BTW ALTER TABLE should be a safe operation to do.
I.e. IF the data could be read by Trino AND Trino allowed  given `ALTER TABLE SET DATA TYPE` THEN the data should still be readable by Trino

Reply:
I borrowed the sentence from Hive documentation. Do you think we should test in all file formats besides ORC?